firebase-firestore/src/test/java/com/google/firebase/firestore/local/LocalStoreTestCase.java

```diff
@@ -912,26 +908,15 @@ public void testPersistsResumeTokens() {
 
     Query query = query("foo/bar");
     int targetId = allocateQuery(query);
-    ByteString resumeToken = resumeToken(1000);
 
-    QueryData queryData = TestUtil.queryData(targetId, QueryPurpose.LISTEN, "foo/bar");
-    TestTargetMetadataProvider testTargetMetadataProvider = new TestTargetMetadataProvider();
-    testTargetMetadataProvider.setSyncedKeys(queryData, DocumentKey.emptyKeySet());
-
-    WatchChangeAggregator aggregator = new WatchChangeAggregator(testTargetMetadataProvider);
-
-    WatchTargetChange watchChange =
-        new WatchTargetChange(WatchTargetChangeType.Current, asList(targetId), resumeToken);
```

Comment:
This is a current change, not a no-change. Why are these are substitutable?

Reply:
Since works because we persist the resume token for both NO_CHANGE and CURRENT:

https://github.com/firebase/firebase-android-sdk/blob/d9ed980c39c58ddf55ab945baa120c2e0a9fa8cb/firebase-firestore/src/main/java/com/google/firebase/firestore/remote/WatchChangeAggregator.java#L109
https://github.com/firebase/firebase-android-sdk/blob/d9ed980c39c58ddf55ab945baa120c2e0a9fa8cb/firebase-firestore/src/main/java/com/google/firebase/firestore/remote/WatchChangeAggregator.java#L138